compiler/src/test/java/com/squareup/anvil/compiler/dagger/InjectConstructorFactoryGeneratorTest.kt

```diff
@@ -1049,6 +1049,167 @@ public class InjectClass_Factory(
     }
   }
 
+  @Test fun `a factory class performs member injection on a super class from another module`() {
+
+    val otherModuleResult = compile(
+      """
+      package com.squareup.test
+
+      import javax.inject.Inject
+
+      abstract class Base {
+
+        @Inject
+        lateinit var base1: List<Int>
+
+        @Inject
+        lateinit var base2: List<String>
+      }
+      """
+    )
```

Comment:
I think it's good practice here to verify that the compilation result is successful nonetheless:
```kotlin
) {
  assertThat(result).isEqualTo(OK)
}
```